sql/src/main/java/org/apache/druid/sql/calcite/aggregation/Aggregations.java

```diff
@@ -48,10 +49,18 @@ private Aggregations()
       final Project project
   )
   {
-    return call.getArgList().stream()
-               .map(i -> Expressions.fromFieldAccess(rowSignature, project, i))
-               .map(rexNode -> toDruidExpressionForSimpleAggregator(plannerContext, rowSignature, rexNode))
-               .collect(Collectors.toList());
+    final List<DruidExpression> args = call
+        .getArgList()
+        .stream()
+        .map(i -> Expressions.fromFieldAccess(rowSignature, project, i))
+        .map(rexNode -> toDruidExpressionForSimpleAggregator(plannerContext, rowSignature, rexNode))
+        .collect(Collectors.toList());
+
+    if (args.stream().noneMatch(Objects::isNull)) {
```

Comment:
Could you add a comment here explaining why we do this part? It seems not obvious to me

Reply:
I added method-level javadocs that explain it:

```java
   * @return list of expressions corresponding to aggregator arguments, or null if any cannot be translated
```